src/components/Person/Person.jsx

```diff
@@ -1 +1,29 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+
+export const Person = ({ person }) => {
+  const { name, age, sex, isMarried, partnerName } = person;
+
+  return (
+    <section className="Person">
+      <h2 className="Person__name">{`My name is ${name}`}</h2>
+      {age
+        && <p className="Person__age">{`I am ${age}`}</p>
+      }
+      {isMarried
+        ? (
+          <p className="Person__partner">
+            {sex === 'f'
+              ? `${partnerName} is my husband`
+              : `${partnerName} is my wife`
+            }
+          </p>
+        )
```

Comment:
It is not a good practice to use ternary operator in ternary operator, maybe you can calculate it from outside?